views/homework/edit.hbs

```diff
@@ -157,7 +157,7 @@
 			<div class="form-group">
 				<label>{{$t "homework.label.task" }}</label>
 				<textarea name="description" type="text" class="form-control customckeditor"
-					placeholder="Aufgabenstellung">{{assignment.description}}</textarea>
+					placeholder="{{$t "homework.label.task" }}">{{assignment.description}}</textarea>
```

Comment:
```suggestion
					placeholder="{{$t "homework.edit.label.task" }}">{{assignment.description}}</textarea>
```
Ist doch nur auf der bearbeiten Seite zu sehen oder?
Oder es müsste mit `homework.global` sein, weil es auf mehreren Seiten sichtbar ist.

Reply:
whoops, dann ist da generell was schief, hab einfach nur nen key übernommen aus dem file...

Reply:
ich führe das fort in der lokalisation rocketchat gruppe

Reply:
lassen wir es für diesen PR erstmal so? ich möchte jetzt nicht alle keys anpassen in dem PR...

Reply:
ja, die keys haben generell nen problem :( Müssen wir nochmal ran, können wir hier skippen.